sentinel2preprocessing/l1c_to_l2a_conversion.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
```

Comment:
Suggestion: shorter name of file: conversion.py. We have docs inside Sentinel2Converter where the aim of class is described

Reply:
Done:
https://github.com/QuantuMobileSoftware/sentinel2tools/pull/7/commits/c652bb43fab7655d8e29a26536d994f883cad3a1